src/components/ComboboxMenu.tsx

```diff
@@ -64,17 +68,26 @@ export const ComboboxMenu = <
     >
       {withSearch && (
         <$Header $withStickyLayout={withStickyLayout}>
-          <$Input
-            /**
-             * Mobile Issue: Search Input will always trigger mobile keyboard drawer. There is no fix.
-             * https://github.com/pacocoursey/cmdk/issues/127
-             */
-            autoFocus
-            value={searchValue}
-            onValueChange={setSearchValue}
-            placeholder={inputPlaceholder}
-            data-hj-allow
-          />
+          {useSearchInputComponent ? (
+            <$SearchInput
+              autoFocus
```

Comment:
hmmm sus on this autofocus here - doesnt seem necessary all of the time

Reply:
it mightttt beee but i'd rather not mess w/ current implementation of a shared component during this rewrite so we can limit surface area of changes

Reply:
i'll add a comment here to look into this

Reply:
actually nvm just gonna remove autofocus. i realized the autofocus is necessary for the other component specifically b/c it's a `Command.Input`